src/Search/Api/Elasticsearch/ElasticQueryBuilder.php

```diff
@@ -66,47 +66,45 @@ private function must($query)
         $this->query['body']['query']['bool']['must'][] = $query;
     }
 
-    public function searchFor(string $string) : QueryBuilder
+    public function searchFor(string $string): QueryBuilder
     {
-        if ($string === '') {
-            $this->sort('_doc');
-        } else {
+        if ($string !== '') {
             $this->query('match', ['_all' => $string]);
         }
 
         return $this;
     }
 
-    public function order(string $direction = 'desc') : QueryBuilder
+    public function order(string $direction = 'desc'): QueryBuilder
     {
         $this->order = $direction === 'desc' ? 'desc' : 'asc';
 
         return $this;
     }
 
-    public function paginate(int $page = 1, int $perPage = 10) : QueryBuilder
+    public function paginate(int $page = 1, int $perPage = 10): QueryBuilder
     {
         $this->query['from'] = ($page - 1) * $perPage;
         $this->query['size'] = $perPage;
 
         return $this;
     }
 
-    public function sortByRelevance($reverse = false) : QueryBuilder
+    public function sortByRelevance($reverse = false): QueryBuilder
     {
-        $this->sort(['_all' => $this->getSort($reverse)]);
+        $this->sort("_score:{$this->getSort($reverse)}");
 
         return $this;
     }
 
-    public function sortByDate($reverse = false) : QueryBuilder
+    public function sortByDate($reverse = false): QueryBuilder
     {
-        $this->sort(['published' => $this->getSort($reverse)]);
+        $this->sort("published:{$this->getSort($reverse)}");
```

Comment:
For now I've added the sorts in the order:

- statusDate (ignored if not there)
- published
- updated (fallback if no published)

So this will hopefully resolve the issue in the short term until the ticket is completed.